### PR TITLE
Added PhotoMessage onError

### PR DESCRIPTION
**Pull request by oxinosg**
* Added PhotoMessage onError

### CODE REVIEW

File: src/MessageBox/MessageBox.js [hunk 1]
```diff
@@ -130,6 +130,7 @@ export class MessageBox extends Component {
                                         onOpen={this.props.onOpen}
                                         onDownload={this.props.onDownload}
                                         onLoad={this.props.onLoad}
+                                        onError={this.props.onError}
```
Comment (abdurrahmanekr) [inline src/MessageBox/MessageBox.js]: Hi @oxinosg, thanks for contribution 🎉Can you change it to `onPhotoError`? If other types of images are added, we may need to change their name. For MessageBox and MessageList

Comment (oxinosg) [reply src/MessageBox/MessageBox.js]: Makes sense :+1:

File: src/PhotoMessage/PhotoMessage.js [hunk 2]
```diff
@@ -41,8 +41,8 @@ export class PhotoMessage extends Component {
                         src={this.props.data.uri}
                         alt={this.props.data.alt}
                         onClick={this.props.onOpen}
-                        onLoad={this.props.onLoad}/>
-
+                        onLoad={this.props.onLoad}
+                        onPhototError={this.props.onPhototError}/>
```
Comment (abdurrahmanekr) [inline src/PhotoMessage/PhotoMessage.js]: @oxinosg just a little typo.

File: src/PhotoMessage/PhotoMessage.js [hunk 3]
```diff
@@ -94,6 +94,7 @@ PhotoMessage.defaultProps = {
     onDownload: null,
     onOpen: null,
     onLoad: null,
+    onPhototError: null,
```
Comment (abdurrahmanekr) [inline src/PhotoMessage/PhotoMessage.js]: And here

**Comment by abdurrahmanekr** [comment]
Thanks again @oxinosg
:tada:

**Comment by oxinosg** [comment]
Thank you for the great work mate :tada:

**Comment by abdurrahmanekr** [comment]
Published [v10.7.0](https://www.npmjs.com/package/react-chat-elements)

**Pull request merged**